app/controllers/todo_controller.rb

```diff
@@ -0,0 +1,3 @@
+class TodoController < ApplicationController
```

Comment:
The rails convention to name controllers is in the plural, so, please rename this as `TodosController` and the filename as `todos_controller.rb`.